installer/install.iss

```diff
@@ -34,6 +33,7 @@
 ; Compiler-related
 Compression=lzma2/ultra64
 LZMAUseSeparateProcess=yes
+LZMANumBlockThreads=2
```

Comment:
I'm not sure if this actually gets close to doubling the speed on GitHub hosted runners.

The docs say:

> Since LZMA2 (and LZMA) uses two threads for match-finding by default (see [CompressionThreads](https://jrsoftware.org/ishelp/topic_setup_compressionthreads.htm)), there ideally should be two processor cores available for each block thread. Thus, to see the maximum benefit from a value of 2, four cores are needed.

Reply:
You're right. When building locally, this was super helpful as I could set it to `LZMANumBlockThreads=2` (my machine has 8 cores), but there likely won't be any performance benefit in CI. Will remove this commit.